specification/cognitiveservices/data-plane/Language/stable/2023-04-01/analyzetext.json

```diff
@@ -315,9 +330,6 @@
           "description": "Parameters for the Abstractive Summarization task."
         }
       },
-      "required": [
-        "parameters"
```

Comment:
Is this change correct?

Reply:
This is an old stable api that we're changing. Was parameters required in this version?

Reply:
This is a bugfix.

Reply:
The swagger is incorrect this is an optional parameter for the service.